app/registration/views/patient-summary.html

```diff
@@ -3,7 +3,7 @@
         <li role="presentation" data-toggle="tab" class="active"><a ui-sref="dashboard.program">{{'COMMON_PROGRAM_ENROLLMENT_TITLE' | translate}}</a></li>
         <li role="presentation" data-toggle="tab" class="active"><a ui-sref="dashboard.visits">{{'COMMON_VISIT_HISTORY_TITLE' | translate}}</a></li>
         <li role="presentation" data-toggle="tab" class="active"><a ui-sref="dashboard.services">{{'COMMON_CLINIC_SERVICES_TITLE' | translate}}</a></li>
-        <li role="presentation" data-toggle="tab" class="active"><a ui-sref="dashboard.alerts">{{'COMMON_ALERT_TITLE' | translate}}</a></li>
+        <!--<li role="presentation" data-toggle="tab" class="active"><a ui-sref="dashboard.alerts">{{'COMMON_ALERT_TITLE' | translate}}</a></li>-->
```

Comment:
@valvijo definately we need to discuss those, but AFAIK, there are only flags for clinician and if there are some for registration, we need to discuss them and figure out how to represent them